### PR TITLE
Use Larger GHA instances from ASWF for Linux/Windows

### PR DESCRIPTION
**Pull request by danrbailey**

### CODE REVIEW

**Comment by danrbailey** [comment]
@idclip - I believe this is ready to be merged now. Apart from the CI runner changes, bumping the number of threads used in GHA and the TBB/Blosc changes from #1481, the only other change was to fix this VDB render CMake issue for static libs on Windows:

https://github.com/AcademySoftwareFoundation/openvdb/pull/1469/files#diff-c7f73fa00a0035a901886287a05980046c7c9c84f8a244bffb7a8895d4b9ce76L74

File: openvdb_cmd/vdb_render/CMakeLists.txt [hunk 1]
```diff
@@ -71,7 +71,11 @@ target_link_libraries(vdb_render
 
 if(WIN32)
   # @note OPENVDB_OPENEXR_STATICLIB is old functionality and should be removed
-  get_target_property(ILMBASE_LIB_TYPE IlmBase::Half TYPE)
+  if (TARGET Imath::Imath)
+    get_target_property(ILMBASE_LIB_TYPE Imath::Imath TYPE)
+  else()
+    get_target_property(ILMBASE_LIB_TYPE IlmBase::Half TYPE)
+  endif()
```
Comment (danrbailey) [inline openvdb_cmd/vdb_render/CMakeLists.txt:78]: @Idclip - any thoughts on this particular change? This is the only bit that doesn't touch the switch to 8 core builds.

**Review by Idclip — APPROVED** [review]

File: openvdb_cmd/vdb_render/CMakeLists.txt (hunk 1 above)
Comment (danrbailey) [reply openvdb_cmd/vdb_render/CMakeLists.txt:78]: I don't know precisely, but I think it's most likely the version of imath/half being provided from vcpkg. It's only failing in the Windows builds.

Comment (e4lam) [reply openvdb_cmd/vdb_render/CMakeLists.txt:78]: The change is consistent with what's needed for upgrading to OpenEXR 3.X when they did the Imath separation. And indeed it looks like vcpkg just updated to OpenEXR 3.1 a mere 12 days ago:
- https://github.com/microsoft/vcpkg/pull/26862

Comment (danrbailey) [reply openvdb_cmd/vdb_render/CMakeLists.txt:78]: Thanks for confirming!

**Pull request merged**